static/flow/util/message-executor.js

```diff
@@ -72,16 +71,12 @@ var MessageExecutor = function(config) {
             this.response_type,
             function(timestamp, params) {
 
-                 console.log("[DEBUG] MessageExecutor handleResponse()",
```

Comment:
At some point, we will need to monitor if there is any slowdown caused by the number of console statements.  The number went up greatly when we started autosaving (since we log the program save state) and historically performance can erode through excessive logging.  Eventually we could wrap all console logs into a method that we turn on or off via command line argument (or even have a brief and verbose mode).